index.fr.md

```diff
@@ -0,0 +1,226 @@
+
+## Les fondamentaux {#basics}
+
+Dans la plupart des cas, inclure les utilisateurs lors de l'évaluation suppose d' :
+
+-   inclure quelques utilisateurs en situation de handicap - et selon votre public cible, des utilisateurs âgés
+-   les inclure tout au long du développement pour effectuer des exemples de tâches sur vos prototypes pour vous permettre d'identifier quels aspects de la conception et du code peuvent être améliorés
+-   échanger à propos des problèmes d'accessibilité avec eux
+
+**Allez sur [[Inclure les utilisateurs dans les projets Web pour une accessibilité plus optimale et plus facile]](/planning/involving-users/)** pour trouver de l'aide dans [Inclure différents utilisateurs](/WAI/users/involving#diverse) et [expérience des utilisateurs interagissant avec le Web](/WAI/users/involving#experience)
+
+Comme pour toute évaluation avec des utilisateurs, inclure des utilisateurs de niveau basique, intermédiaire ou avancé, dépendra de vos utilisateurs cibles. Par exemple, si vous développez une application Web pour des comptables au sein d'une entreprise, vous opterez certainement pour des utilisateurs de technologies d'assistance d'un niveau avancé. Par contre, pour un site Web destiné aux demandes de prestation d'invalidité, vous travaillerez plutôt avec des utilisateurs de technologies d'assistance novices.
+
+<mark><strong>Attention :</strong> prêtez attention à toutes les données. <strong>Évitez de supposer que les données d'une personne présentant une certaine forme de handicap s'appliquent à toutes les personnes handicapées. </strong></mark>
+
+Une personne handicapée ne sait pas forcément comment d'autres personnes présentant le même type de handicap interagissent avec le Web. Elle pourrait ne pas en savoir assez sur les autres handicaps pour fournir une aide valable pour d'autres questions d'accessibilité. Il est préférable de rassembler des données de plusieurs types d'utilisateurs.
+
+Note : en plus de détecter des problèmes d'accessibilité, l'évaluation menée avec des utilisateurs handicapés révèlent des problèmes généraux d'utilisation qui touchent tous les utilisateurs, y compris ceux qui ne se trouvent *pas* en situation de handicap.
+
+## Analyse des problèmes d'accessibilité {#analyz}
+
+L'accessibilité Web dépend de l'association [des différents composants du développement Web avec l'interaction](/fundamentals/components/), dont les navigateurs Web, les technologies d'assistance, et le contenu Web.
+
+Les problèmes d'accessibilité peuvent être causés par un ou plusieurs composants. Par exemple, si un utilisateur qui ne peut pas utiliser une souris n'arrive pas à accéder aux fonctionnalités clavier des technologies d'assistance, il peut y avoir plusieurs explications :
+
+-   le développeur n'a pas balisé/codé correctement la page Web, ou
+-   le navigateur ou le lecteur multimédia ne lit pas correctement le code, ou
```

Comment:
The original version uses "markup". "balisage" would therefore be more appropriate than "code" ?